lib/puppet/ssl/host.rb

```diff
@@ -522,6 +588,7 @@ def ensure_ca_certificate
       end
     end
   end
+  public :ensure_ca_certificate
```

Comment:
Should we just move this and `check_for_certificate_on_disk` out of the private section? I've heard varying opinions on using tags versus the single `private` keyword, but mixing them seems not great.

Reply:
I generally try to keep things either above or below the private line, but in this case I wanted to minimize the rebase conflicts you would have. Happy to make the change if you want?